README.md

```diff
@@ -0,0 +1,22 @@
+# ICCOPT 2022 JuMP Tutorial
+
+This repository will contain materials for the Julia/JuMP tutorial at
+[ICCOPT 2022](https://iccopt2022.lehigh.edu/summer-school/summer-school-program/).
+
+## Requirements
+
+Participants are expected to have their own laptop to follow along with the tutorial.
+
+Please install:
+- Julia v1.6 or v1.7 from https://julialang.org/downloads/.
+- [Visual Studio Code](https://code.visualstudio.com/) (text editor) with [Julia extensions](https://code.visualstudio.com/docs/languages/julia)
+
+### Common issues
+
+See Julia's
+[platform-specific instructions](https://julialang.org/downloads/platform/#platform_specific_instructions_for_official_binaries)
+for more guidance.
+
+If you have trouble installing before the tutorial, please
+[open a GitHub issue](https://github.com/blegat/TraDE-OPT_2022/issues/new) and we'll try to help.
```

Comment:
Wrong link?

Reply:
Good catch, maybe I was sending everyone to Benoit for help :)